tailwind.config.ts

```diff
@@ -14,6 +14,9 @@ const config = {
         white: "#fffceb",
         red: "#E6002A"
       },
+      screens: {
+        'tall': { 'raw': '(min-width: 1024px) and (min-height: 700px)' }
+      }
```

Comment:
I added this because since I'm setting `h-[90vh]`, it would always set the height to 90% of the screen height, but that would mess up the layout when I e.g. opened devtools, which makes the screen height very small. `min-h-[90vh]` doesn't work because the canvas is pretty tall and it was still taking up >95% of the height in that case. Hence, the new `tall` breakpoint is the Tailwind `lg` breakpoint except also with a min height for the screen, so if devtools are open it lets the hero be the size it needs to be in order for the content to fit.